modules/User.php

```diff
@@ -1,6 +1,108 @@
 <?php
 	class User {
-		function __construct() {
-			return;
+		public $email;
+		public function __construct($email) {
+			if (Utils::isValidEmail($email)) {
+				$this->email = $email;
+			} else {
+				return NULL;
+			}
+		}
```

Comment:
https://github.com/systers/FirstAide-web/pull/232#discussion_r120679353 Explained it here.